feat: Start associating packaging shapes images in the taxonomy

teolemon opened the pull request:
### What
- Start associating packaging shapes images in the taxonomy
- Sometimes we have several shapes per entry:
  - in that case I've tried to take the most representative
  - in some cases, create subentries that are currently commented out
- I have added suggested materials per the official CITEO guidelines
  - They could be displayed on top of material suggestion lists, and possibly used for potential Robotoff questions

teolemon commented on taxonomies/packaging_shapes.txt:
```diff
@@ -138,9 +139,13 @@ hu:Doboz, dobozok
+# suggest:packaging_materials:nl: en:paper, en:cardboard
 description:nl:Een veelal kartonnen balkvormige verpakking
 
+fr:box de pâtes
```
```suggestion
# <en:box
# fr:box de pâtes
```

teolemon commented on taxonomies/packaging_shapes.txt:
```diff
@@ -151,8 +156,15 @@ pl:wytłaczanka
 description:nl:Kartonnen of geschuimd kunststoffen doos met holtes om eieren in te verpakken zonder dat deze elkaar raken
+# image:fr:boite-boitier-4.svg
+
+<en:Box
+fr:Boîte à hamburger
+# suggest:category: en:burgers
```
```suggestion
# <en:Box
# fr:Boîte à hamburger
# suggest:category: en:burgers
```

teolemon commented on taxonomies/packaging_shapes.txt:
```diff
@@ -188,16 +202,58 @@ it:Bottiglia
+# image:fr:bouteille-2.svg
+# suggest:category: en:non-sparkling wines
+
+fr:bouteille de jus
```
```suggestion
# fr:bouteille de jus
```

teolemon commented on taxonomies/packaging_shapes.txt:
```diff
@@ -188,16 +202,58 @@ it:Bottiglia
+# image:fr:bouteille-6.svg
+# suggest:category: en:juices
+
+#fr:bouteille de bière
```
```suggestion
# fr:bouteille de bière
```

teolemon commented on taxonomies/packaging_shapes.txt:
```diff
@@ -188,16 +202,58 @@ it:Bottiglia
+# suggest:category: en:juices
+
+#fr:bouteille de bière
+# image:fr:bouteille-1.svg
+# suggest:category: en:beers
+
+#fr:grande bouteille de soda
+# image:fr:bouteille-7.svg
+# suggest:category: en:sodas
+
+#fr:bouteille compactable
+# image:fr:bouteille-3.svg
+# suggest:category: en:waters
+
+#fr:bouteille de Javel
+# image:fr:bouteille-8.svg
+# suggest:category: en:detergeants
+
+#fr:bouteille de liquide vaisselle
+# image:fr:bouteille-9.svg
+# suggest:category: en:detergeants
+
+
+
+fr:flasque
```
```suggestion
# image:fr:bouteille-1.svg
# suggest:category: en:beers

# <en:bottle
# fr:grande bouteille de soda
# image:fr:bouteille-7.svg
# suggest:category: en:sodas

# <en:bottle
# fr:bouteille compactable
# image:fr:bouteille-3.svg
# suggest:category: en:waters

# <en:bottle
# fr:bouteille de detergeant
# image:fr:bouteille-8.svg
# suggest:category: en:detergeants

# <en:bottle
# fr:bouteille pour liquide vaisselle
# image:fr:bouteille-9.svg
# suggest:category: en:detergeants



fr:flasque
```

teolemon commented on taxonomies/packaging_shapes.txt:
```diff
@@ -188,16 +202,58 @@ it:Bottiglia
 nl:Drankkarton, Melkpak, Drankenkartons, Drankencarton, Drankenkarton, Drinkpahk, Drinkpakken
 description:en:Verpakking voor vloeibare levensmiddelen gemaakt van gelamineerd papier
 
+fr:brique à ouvrir par découpe
+# image:fr:brique-1.svg
+# suggest:category: en:milks
+
+fr:brique biseautée avec bouchon revissable
+# image:fr:brique-2.svg
+
+fr:brique plate avec bouchon revissable
+# image:fr:brique-3.svg
+
```
```suggestion
# <en:Brick
# fr:brique à ouvrir par découpe
# image:fr:brique-1.svg
# suggest:category: en:milks

# <en:Brick
# fr:brique biseautée avec bouchon revissable
# image:fr:brique-2.svg

# <en:Brick
# fr:brique plate avec bouchon revissable
# image:fr:brique-3.svg

```

teolemon commented on taxonomies/packaging_shapes.txt:
```diff
@@ -277,6 +344,17 @@ hu:Tubus, tubusok, Cső, csövek, cső alakú
 describtion:nl:Een verpakking in kokervorm gemaakt van metaal of plastic gevuld met een halfvloeibare stof
+# image:fr:tube-1.svg 
+# suggest:packaging_materials: en:cardboard, en:metal, en:glass, en:plastic, en:others
+#associated_packagings: en:cap
```
```suggestion
# associated_packagings: en:cap
```

teolemon commented on taxonomies/packaging_shapes.txt:
```diff
@@ -535,13 +666,15 @@ pt:Película, películas, filme, filmes, filme extensível, filme estirável
 description:en:A thin layer of plastic or metal
 description:nl:Dun en buigzaam vel van een metaal of kunststof
-#comment:en:Some languages might distinghuish between plastic film and metal foil. Film in english can also mean a thin sheet of oil on water for instance.
+#comment:en:Some languages might distinghuish between plastic film and metal foil. Film in English can also mean a thin sheet of oil on water for instance.
```
```suggestion
# comment:en:Some languages might distinguish between plastic film and metal foil. Film in English can also mean a thin sheet of oil on water for instance.
```

teolemon commented on taxonomies/packaging_shapes.txt:
```diff
@@ -611,149 +759,245 @@ hu:címke, címkék
 fr:Bag-in-box, Cubitainer
 nl:Zak-in-doos
 wikidata:en:Q799810
+# made of: Boîte carton à recycler, poche plastique à jeter
```
```suggestion
# made of: fr:Boîte carton à recycler, fr:poche plastique à jeter
```

teolemon commented:
@aleene @stephanegigandet would like your input

stephanegigandet commented:
> @aleene @stephanegigandet would like your input

It's a great idea to add images for each packaging shape.

Regarding the format of the property:
- we should use image:xx: instead of image:en: so that the images apply to all languages
- we should move the CITEO images to from html/images/icons/packaging-shapes/ to html/images/icons/packaging-shapes/citeo/ so that we can easily add other images

so the format should be something like:

image:xx:/images/icons/packaging-shapes/citeo/brique-2.svg

Regarding the content:

We may not need to add all Citeo icons as entries in the taxonomy. If we do add them, then we should have an English names for them. It might be useful to have all of them at some point (e.g. to recognize them, even though my understanding is that they will go away in France).

The focus at first should be on getting an image for all existing entries (or their parents, we can inherit the property), so that we can use them in the packaging edit interface and in knowledge panels etc.
e.g. We can assign the most generic looking bottle image to en:bottle.

teolemon commented:
@stephanegigandet 👍 
- fixed the xx lang code
- moved the images to a dedicated CITEO directory
- everything remains commented out for the time being, and we can uncomment as we feel confortable

stephanegigandet reviewed (approved):
Merging as-is, we can decide on the formatting, add image paths etc. after.
